network/horizon/horizon-providers.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 70
+title: Ecosystem Horizon Providers
+---
+
+## Ecosystem Horizon Providers
+
+Multiple infrastructure providers have made Horizon services available, and offer plans ranging from free to high throughput endpoints. These providers can be used for development, testing, and production.
+
+These providers allow access to the Testnet and Mainnet networks.
+
+| Provider                                                                           | Access                                              | Futurenet     | Testnet   | Mainnet     |
+| ---------------------------------------------------------------------------------- | --------------------------------------------------- | ------------- | --------- | ----------- |
+| [Blockdaemon\*](https://docs.blockdaemon.com/reference/supported-stellar-methods) | [Access](https://www.blockdaemon.com/apply/soroban) | Not available | Available | Available   |
```

Comment:
Their landing page may be a little confusing since it feels like it's setting you up for just an RPC node, but not sure if we really have any control over that.

Reply:
I could send them an email about it.